MessengerClone/ChatClient.swift

```diff
@@ -16,7 +16,7 @@ extension ChatClient {
         components.navigation.channelListRouter = MessengerChatChannelListRouter.self
         appearance.images.pencil = UIImage(systemName: "square.and.pencil")!
         
-        components.channelList.itemView = ChatChannelListItemView.SwiftUIWrapper<MessengerChatChannelListItem>.self
+        components.channelList.channelListItemView = ChatChannelListItemView.SwiftUIWrapper<MessengerChatChannelListItem>.self
```

Comment:
This is probably not compiling right ? 🤔 channelList does not exist anymore

Reply:
We can also put back CI actions that check the sample apps are buildable

Reply:
Let's not worry about the clone app for now please.